content/en/posts/html/responsive-dom/index.md

```diff
@@ -0,0 +1,327 @@
+---
+date: "2013-12-26"
+title: Responsive webdesign and DOM structure
+tags:
+  - responsive
+  - html
+  - javascript
+authors:
+  - neemzy
+---
+
+Alright, first things first, if you don't mind. Ever heard about **responsive
+webdesign**? Yes? Good. You therefore may have encountered some pretty annoying
+use case (I did). Whether you are a *mobile-first* adept or an old-fashioned
+progressive degradator, you must have come to a point where you told yourself
+this silly thing:
+
+> Would really be great if this element was here instead of there. But it
+> isn't.
+
+Your typical example for this is a slightly complex menu, where we would like
+to give the items a bit of reordering just above or below a certain screen
+size. And we have to admit there is no miraculous way out of this.
+
+I can already hear the nasty boys scream and shout, saying that a better
+conception beforehand would have granted us better *markup* and spared us the
+trouble. Although it's an easy thing to say, it turns out to be true in a
+number of cases (yes, I'm quite good at avoiding picking a side). However, if
+you ever find yourself wading in these swamps, it probably means going back
+isn't an option anymore.
+
+As fast as innovation in our profession goes, especially about anything
+mobile-related in general and web-related especially, it's quite likely that
+the tricks I'm going to deliver are obsolete six months from now, and that
+this article becomes increasingly popular again on Twitter, RTs kindly mocking
+the pre-Columbian techniques we were limited to not so long ago. Heck knows, I
+might just become a paradoxally decadent celebrity *à la* Rick Astley. In the
+meantime, I have nothing better to share on the subject. If you do, I strongly
+recommend spitting it out in the comments for everyone to enjoy. Well, are you
+ready now? I'm waiting, you know!
+
+## Vintage: Do Repeat Yourself
+
+Consider a blog page, where successively appear the article's title, its
+photo, and its text at last. Given that a photo, depending your connection's
+quality, can be quite a PITA to load on mobile (and that the responsive image
+thing [est encore un sacré bordel](is still a goddamn mess)), you decide to
```

Comment:
Ah, my bad. No, it wasn't, I just messed up. It's fixed now, thanks for the highlight! :)